app/routes/sprint-7/caseworkerManageRoutes.js

```diff
@@ -540,6 +502,49 @@ router.post(
   }
 );
 
+const assessmentJourney = ["attended", "safeguarding", "needs", "circumstances", "outcomes", "metrics", "details"];
+
+for (var i = 0; i < assessmentJourney.length; i++) {
```

Comment:
Is there a reason you've done an old fashioned `for` loop, rather than a `for...of` (nicer to read):
```
for (let page of assessmentJourney) {}
```
?

(same below)

Reply:
For one of the loops (the one that handles POST requests) I needed the loop index in order to get the _next_ element from the array. Is there a nicer way to do that? The other one I don't need it, just kept 'em the same for consistency.

Reply:
Ah, I knew I'd missed something but didn't think to look too closely at the second loop 🤦 

No, I think you're right. Although we tend to use `let` rather than `var` now in JS.

Reply:
Ah dammit, I always get these keywords confused with their meanings in Swift, I'll change it